src/Dynamo.All.sln

```diff
@@ -217,284 +217,567 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Notifications", "Notificati
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Workspaces", "Workspaces\Workspaces.csproj", "{D2AAA2DD-6C7E-41F4-8E36-8E0D112CB8DC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPackagesUI", "DynamoPackagesUI\DynamoPackagesUI.csproj", "{297F6290-0325-4C9B-9E77-DB9041EB61F3}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
 		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Debug|x64.Build.0 = Debug|Any CPU
 		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Release|Any CPU.Build.0 = Release|Any CPU
-		{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7858FA8C-475F-4B8E-B468-1F8200778CF8}.Release|x64.ActiveCfg = Release|Any CPU
```

Comment:
We don't need these changes. You can simply delete x64 configuration for the solution using configuration manager.